httpbakery/client.go

```diff
@@ -338,10 +345,18 @@ func NewCookie(ms macaroon.Slice) (*http.Cookie, error) {
 // contain a single primary macaroon in its first element, and any
 // discharges after that.
 func SetCookie(jar http.CookieJar, url *url.URL, ms macaroon.Slice) error {
+	return SetCookieWithPath(jar, url, "", ms)
+}
+
+// SetCookieWithPath is like SetCookie except the cookie path is
```

Comment:
I'm not sure that this pulls its weight. I'd just document in SetCookie that if you want to set the cookie for a particular path or with a given name, then use NewCookie, set the relevant fields and then use CookieJar.SetCookies.